Automattic/newspack-newsletters
# Add Posts Inserter to modal and strip placeholder style

### All Submissions:






### Changes proposed in this Pull Request:

* Adds Posts Inserter to the Daily/Weekly layout.
* Removes the Posts Inserter placeholder style so preview looks more inline.

__Before:__

![1 before](https://user-images.githubusercontent.com/177929/79861073-b23a8200-83cb-11ea-829b-3d6b0c035c8c.png)

__After:__

![2 after](https://user-images.githubusercontent.com/177929/79861089-b8306300-83cb-11ea-90df-7180869509f7.png)

### How to test the changes in this Pull Request:

1. Add new
2. Preview the Daily/Weekly layout. Do you see the posts without the placeholder border style?
3. Select layout
4. You should see the "regular" Posts Inserter now.

### Other information:
